docs/src/main/tut/monad.md

```diff
@@ -7,3 +7,98 @@ scaladoc: "#cats.Monad"
 ---
 # Monad
 
+Monad extends the Applicative typeclass with a new function `flatten`. Flatten
+takes a value in a nested context (eg. `F[F[A]]` where F is the context) and
+"joins" the contexts together so that we have a single context (ie. F[A]).
+
+The name `flatten` should remind you of the functions of the same name on many
+classes in the standard library.
+
+```tut
+Option(Option(1)).flatten
+Option(None).flatten
+List(List(1),List(2,3)).flatten
+```
+
+### Monad instances
+
+If Applicative is already present and `flatten` is well-behaved, extending to
+Monad is trivial. The catch in cats' implementation is that we have to override
+`flatMap` as well.
+
+`flatMap` is just map followed by flatten.
+
+```tut
+import cats._
+
+implicit def optionMonad(implicit app: Applicative[Option]) =
+  new Monad[Option] {
+  override def flatten[A](ffa: Option[Option[A]]): Option[A] = ffa.flatten
+  override def flatMap[A, B](fa: Option[A])(f: A => Option[B]): Option[B] =
+    app.map(fa)(f).flatten
+  // Reuse this definition from Applicative.
+  override def pure[A](a: A): Option[A] = app.pure(a)
+}
```

Comment:
Sorry for the nitpick. Could you indent the code within the `new Monad[Option] {` block?